src/sentry/static/sentry/app/components/modals/inviteMembersModal/index.tsx

```diff
@@ -368,12 +369,8 @@ class InviteMembersModal extends AsyncComponent<Props, State> {
           />
         ))}
 
-        <AddButton
-          disabled={disableInputs}
-          priority="link"
-          onClick={this.addInviteRow}
-          icon="icon-circle-add"
-        >
+        <AddButton disabled={disableInputs} priority="link" onClick={this.addInviteRow}>
+          <StyledIconAdd size="xs" circle />
```

Comment:
That's the thing - I'm hoping we don't have to add margins when Icons are added to a Button. As soon as the migration is complete, we can styled any svg in the button component to have a right margin.

Reply:
I think it'd be better to add icons via props, e.g. if we had a right aligned button we wouldn't want a right margin

Reply:
![Screen Shot 2020-02-24 at 12 42 43 PM](https://user-images.githubusercontent.com/4830259/75189541-40172b00-5703-11ea-9f05-473aaa0f33c5.png)

@billyvg We currently have `padding: 12px 16px`. If we evenly distribute this amongst the children it should be fine - 12px on the border and 4px on children. We can do this on all variations

Reply:
@doralchan How would we go about doing this in `<Button>`, if the goal is for users to not need to add custom margin/padding to icons?

Reply:
Adding 4px side padding to any button children could work

Reply:
Having used the new buttons a bit. It is a bit tedious to have to create a styled component for each icon that needs to live inside a Button. If the margins could be automatically applied that would be simpler to use an create more consistent spacing.

Reply:
I think we'd have more flexibility down the road if we defined icons through a prop, otherwise it's a bit difficult to search for an icon inside of its children.